jobs/collect_logs.sh

```diff
@@ -3,7 +3,7 @@ source /usr/local/src/cinder-ci/jobs/utils.sh
 
 echo "Collecting logs"
 
-if [ $DEBUG_JOB = "yes" ]; then
+if [ $DEBUG_JOB = "yes" ] || [ -z '$DEBUG_JOB' ]; then
```

Comment:
-bash: [: =: unary operator expected
Rings any bells on "="?

On the other hand, if DEBUG_JOB is not defined I guess it's considered not set, not equal to "yes"!